retro-board-app/src/translations/languages.ts

```diff
@@ -109,6 +109,11 @@ export default [
     name: 'Português Brasileiro',
     englishName: 'Portuguese (Brazilian)',
   },
+<<<<<<< HEAD
```

Comment:
Hi! Thanks so much for your contribution. You seem to have a merge issue on that file, can you resolve it please? this file shouldn't have changed at all since the Italian translation was already there. Thanks!

Reply:
Hi @antoinejaussoin, I resolve the conflict and it seems everything's ok ("all checks have passed"). Let me know if it's ok or I need to work on it again.
Thanks!

Reply:
Thanks a lot for your work :) All merged!

Reply:
Always glad to help! :)